### PR TITLE
Fix various Issues for 1.7

### LINKED ISSUE #972: Language with same name issue
"English" and "Deutch" are linked with 2 different flags (American english and UnitedKingdom english for example). When we choose american english (with american flag) in the config panel, when we come back in the config it will display the UnitedKingdom flag (because it is the first "english" flag in the list

### PR DESCRIPTION
**Pull request by kuhlkrein**
Should fix #972, #992 and #994

### CODE REVIEW

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/multilinguism/Languages.java [hunk 1]
```diff
@@ -44,45 +44,45 @@
...
-        put(result, build(new LanguageLocale("pol"), "polszczyzna", false, true, "800px-Flag_of_Poland.svg.png"));
-        put(result, build(new LanguageLocale("por","PT"), "Português", false, true, "800px-Flag_of_Portugal.svg.png"));
-        put(result, build(new LanguageLocale("por","BR"), "Português", false, true, "800px-Flag_of_Brazil.svg.png"));
-        put(result, build(new LanguageLocale("nld"), "Nederlands", false, true, "800px-Flag_of_the_Netherlands.svg.png"));
-        put(result, build(new LanguageLocale("rus"), "Pу́сский язы́к", false, true, "800px-Flag_of_Russia.svg.png"));
-        put(result, build(new LanguageLocale("spa"), "Español", false, true, "800px-Flag_of_Spain.svg.png"));
-        put(result, build(new LanguageLocale("vnm"), "Tiếng Việt", false, true, "800px-Flag_of_Vietnam.svg.png"));
-        put(result, build(new LanguageLocale("zsm"), "Bahasa Melayu", false, true, "800px-Flag_of_Malaysia.svg.png"));
+    private static SortedMap<String, LanguageDetails> createLanguagesMap() {
+        SortedMap<String, LanguageDetails> result = new TreeMap<>();
```
Comment (YannRobert) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/multilinguism/Languages.java]: Pass a Comparator<Locale> to the TreeMap constructor
Use Locale as key of the SortedMap

Comment (kuhlkrein) [reply gazeplay-commons/src/main/java/net/gazeplay/commons/utils/multilinguism/Languages.java]: Done, that's way better, thank you, I didn't know about this constructor :+1:

**Comment by kuhlkrein** [comment]
./gradlew works fine on my local version (BUILD SUCCESSFUL) , do I need to add some special parameter to see the problem ?

**Comment by kuhlkrein** [comment]
Ok I restarted the build and now it seems to work... the first time it told me it was unable to find findbugs or spotbugs jar, don't remember wich one

**Comment by Tohaker** [comment]
>  the first time it told me it was unable to find findbugs or spotbugs jar, don't remember wich one

If that happens again you can just run `./gradlew --refresh-dependencies` and it will redownload all the resources from scratch.

**Comment by kuhlkrein** [comment]
@Tohaker ok Miles thank you !

**Pull request merged**